src/Scoper/StringReplacer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the humbug/php-scoper package.
+ *
+ * Copyright (c) 2017 Théo FIDRY <theo.fidry@gmail.com>,
+ *                    Pádraic Brady <padraic.brady@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Humbug\PhpScoper\Scoper;
+
+use Humbug\PhpScoper\Scoper;
+use Humbug\PhpScoper\Throwable\Exception\RuntimeException;
+
+final class StringReplacer implements Scoper
+{
+    /**
+     * @var array
+     */
+    private $replaceMap;
```

Comment:
the default value could be an empty array to avoid the [`is_array($this->replaceMap`](https://github.com/humbug/php-scoper/pull/65/files#diff-116fe39c87c6067b54d0c5cb7b90f772R48) and remove the `@var` (as the type would become clear from the declaration)